Source/WebCore/page/Quirks.h

```diff
@@ -98,6 +98,7 @@ class Quirks {
     WEBCORE_EXPORT bool needsYouTubeMouseOutQuirk() const;
 
     WEBCORE_EXPORT bool shouldAvoidUsingIOS13ForGmail() const;
+    WEBCORE_EXPORT bool shouldDisableWritingSuggestionsByDefaultQuirk() const;
```

Comment:
Nit: this is a method on the `Quirks` class anyway, so a `Quirk` suffix here seems redundant.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/0d0f6d87804037f405b74be05b723516f4c1268c